src/Microsoft.DotNet.Build.Tasks.Feed/Microsoft.DotNet.Build.Tasks.Feed.csproj

```diff
@@ -20,18 +20,19 @@
     <StartupObject />
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="..\Microsoft.DotNet.Build.CloudTestTasks\AzureBlobLease.cs" />
-    <Compile Include="..\Microsoft.DotNet.Build.CloudTestTasks\AzureConnectionStringBuildTask.cs" />
-    <Compile Include="..\Microsoft.DotNet.Build.CloudTestTasks\AzureHelper.cs" />
-    <Compile Include="..\Microsoft.DotNet.Build.CloudTestTasks\ListAzureBlobs.cs" />
-    <Compile Include="..\Microsoft.DotNet.Build.CloudTestTasks\UploadClient.cs" />
     <Compile Include="BlobFeed.cs" />
     <Compile Include="BlobFeedAction.cs" />
     <Compile Include="ConfigureInputFeed.cs" />
     <Compile Include="PushToBlobFeed.cs" />
+    <Compile Include="SleetLogger.cs" />
+    <Compile Include="SleetSettings.cs" />
+    <Compile Include="SleetSource.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="PackageFiles\**\*.*" />
+    <None Include="PackageFiles\Publish.targets">
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="PackageFiles\Restore.targets" />
```

Comment:
I think you should keep the `PackageFiles\**\*.*` rather than listing each one. (I know VS sometimes likes to expand it.)